src/pages/Stream/hooks/useParamsController.ts

```diff
@@ -54,15 +58,17 @@ const storeToParamsObj = (opts: {
 	page: string;
 	rows: string;
 	query: string;
+	filterType: string;
 }): Record<string, string> => {
-	const { timeRange, offset, page, view, rows, query } = opts;
+	const { timeRange, offset, page, view, rows, query, filterType } = opts;
 	const params: Record<string, string> = {
 		...deriveTimeRangeParams(timeRange),
 		view,
 		offset,
 		rows,
 		page,
 		query,
+		filterType,
```

Comment:
we can add a condition here to set the filter type in the url params when there is a valid query